### PR TITLE
[14.0][ADD]  account_invoice_mode_daily

### PR DESCRIPTION
**Pull request by mt-software-de**
Adding module for automated daily invoicing.
~Depends on: https://github.com/OCA/account-invoicing/pull/1598~

### CODE REVIEW

**Comment by mt-software-de** [comment]
@jbaudoux @etobella @kluna1998 do you have time for a review?

**Comment by mt-software-de** [comment]
@etobella and @pedrobaeza since you approved and reviewed https://github.com/OCA/account-invoicing/pull/1598. Could you please review this one?

**Comment by pedrobaeza** [comment]
Sorry, that one was to unblock the situation, but now you should find other reviewers. You can review other PRs, and ask in exchange that they review yours.

**Review by vuwnevska — APPROVED** [review]
Looks good

File: account_invoice_mode_daily/data/queue_job_data.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- Queue Job Channel -->
+    <record id="invoice_weekly" model="queue.job.channel">
+        <field name="name">invoice_weekly</field>
```
Comment (jbaudoux) [inline account_invoice_mode_daily/data/queue_job_data.xml]:
```suggestion
    <record id="invoice_daily" model="queue.job.channel">
        <field name="name">invoice_daily</field>
```

File: account_invoice_mode_daily/data/queue_job_data.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        <field name="parent_id" ref="queue_job.channel_root" />
+    </record>
+
+    <!-- Queue Job Function -->
+    <record id="job_function_generate_invoices_by_partner" model="queue.job.function">
+        <field name="model_id" ref="sale.model_sale_order" />
+        <field name="method">_generate_invoices_by_partner</field>
+        <field name="channel_id" ref="invoice_weekly" />
```
Comment (jbaudoux) [inline account_invoice_mode_daily/data/queue_job_data.xml]:
```suggestion
        <field name="channel_id" ref="invoice_daily" />
```

**Review by jbaudoux — COMMENTED** [review]
wrong copy / paste?

**Comment by mt-software-de** [comment]
> wrong copy / paste?

yes thx.

**Review by jbaudoux — APPROVED** [review]

**Review by etobella — APPROVED** [review]
/ocabot merge noibump

**Pull request merged**